hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -285,6 +285,24 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	// Set ValidConfiguration condition
+	{
+		condition := metav1.Condition{
+			Type:               string(hyperv1.ValidHostedClusterConfiguration),
+			ObservedGeneration: hcluster.Generation,
+		}
+		if err := r.validateConfigAndClusterCapabilities(hcluster); err != nil {
+			condition.Status = metav1.ConditionFalse
+			condition.Message = err.Error()
+			condition.Reason = "InvalidConfiguration"
```

Comment:
Please move `InvalidConfiguration` into an API constant `InvalidConfigurationReason`

Reply:
Done

Reply:
It's still a string here though

Reply:
@rmohr I think you forgot change to use the constant here, other than that lgtm

Reply:
Oh true. I "fixed" it on the other controller. Now both use the constant.